include/z64actor.h

```diff
@@ -460,16 +478,16 @@ typedef enum {
 #define ACTOR_FLAG_40            (1 << 6)
 // Invisible
 #define ACTOR_FLAG_80            (1 << 7)
-// Related to talk
-#define ACTOR_FLAG_100           (1 << 8)
+// Player has requested to talk to the actor; Player uses this flag differently than every other actor
+#define ACTOR_FLAG_TALK_REQUESTED (1 << 8)
 // 
 #define ACTOR_FLAG_200           (1 << 9)
 // 
 #define ACTOR_FLAG_400           (1 << 10)
 // 
 #define ACTOR_FLAG_800           (1 << 11)
-// 
-#define ACTOR_FLAG_1000          (1 << 12)
+// Actor will not shake when a quake occurs
+#define ACTOR_FLAG_IGNORE_QUAKE  (1 << 12)
```

Comment:
This is also documented in that other quake PR, but I'm not worried if it's also included here